virttest/utils_sys.py

```diff
@@ -72,3 +72,31 @@ def get_host_bridge_id(session=None):
 
     host_bridges = re.findall(hostbridge_regex, output)
     return host_bridges if host_bridges else []
+
+
+def get_pids_for(process_names, session=None):
+    """
+    Given a list of names, retrieve the PIDs for
+    matching processes. Sort of equivalent
+    to: 'ps aux | grep name'
+
+    :param process_names: List of process names to look for
+    """
+
+    status, ps_cmd = cmd_status_output("ps aux", shell=True, session=session)
+    if status != 0 or not ps_cmd:
+        return []
+
+    ps_output = ps_cmd.split("\n")
+    relevant_procs = [
+        proc
+        for proc in ps_output
+        for wanted_name in process_names
+        if wanted_name in proc
+    ]
+
+    relevant_procs = [line.split() for line in relevant_procs]
+    relevant_pids = [int(proc[1]) for proc in relevant_procs]
+    relevant_pids.sort()
```

Comment:
If we sort the pids forcefully,  those pids could not match those process names in their original order.
The order might be useful for later use. So I suggest we'd better provide an option for sorting. 
What do you think of?